include/ck_tile/host/arg_parser.hpp

```diff
@@ -177,6 +229,7 @@ class ArgParser
 
     float get_float(const std::string& name) const
     {
+        assert(("The param contians invalid number", ValidNumber::isNumber(get_str(name))));
```

Comment:
Let's not be so pedantic. This is redundant. And moreover the `assert` function is no-op in Release build so it won't be checked anyway.